kuma/javascript/src/banners.jsx

```diff
@@ -189,20 +200,43 @@ export default function Banners(props: BannersProps) {
         () => [
             {
```

Comment:
Given the static nature of this information I'd recommend moving it outside of the component fn and to the top-level of this module.